src/Map/ProspectorOverlayLayer.cs

```diff
@@ -306,19 +403,28 @@ private void OnChatMessage(int groupId, string message, EnumChatType chattype, s
             if (pos == null || groupId != GlobalConstants.InfoLogChatGroup || !_triggerwords.Any(triggerWord => message.StartsWith(triggerWord)))
                 return;
 
-            message = _cleanupRegex.Replace(message, string.Empty);
             var posX = pos.X / _chunksize;
             var posZ = pos.Z / _chunksize;
-            _messages.RemoveAll(m => m.X == posX && m.Z == posZ);
-            _messages.Add(new ProspectInfo(posX, posZ, message));
-            _clientApi.SaveDataFile(Filename, _messages);
+            _prospectInfos.RemoveAll(m => m.X == posX && m.Z == posZ);
+            _prospectInfos.Add(new ProspectInfo(posX, posZ, message));
+            _clientApi.SaveDataFile(Filename, _prospectInfos);
```

Comment:
Not sure it should save everytime, it's ok for now but maybe add a TODO to offload this to some timer based logic, also not sure how big the file needs to be to be slow from a user perspective (maybe sqlite can be leveraged)